h/static/styles/partials/_form-input.scss

```diff
@@ -149,6 +150,24 @@
     border-radius: 3px;
   }
 
+  // Set the padding depending on whether or not this field has an associated
+  // label. The `<label>` lives outside the input in the DOM but is positioned
+  // _inside_ the `<input>`'s border visually.
+  //
+  // We do this rather than use a separate element for the field's border so
+  // that clicking anywhere inside the visual border focuses the input.
+  .form-input__input.has-label {
+    padding-top: $top-padding;
+    padding-bottom: $bottom-padding;
+    --bottom-padding: $bottom-padding;
```

Comment:
If you haven't seen it before this syntax "declares" a CSS variable, which is then used later on with `var(--bottom-padding)` when computing the new padding after applying a thicker border.